code/modules/mob/living/silicon/robot/robot_vr.dm

```diff
@@ -142,6 +142,10 @@
 
 /mob/living/silicon/robot/proc/vr_sprite_check()
 	if(wideborg == TRUE)
+		if(icontype== "Drake") // Why, Why can't we have normal nice things
+			icon = 'icons/mob/drakeborg/drakeborg_vr.dmi' 
+		else
+			icon = 'icons/mob/widerobot_vr.dmi'
```

Comment:
This else is not really needed, it worked without that

Reply:
the else is for the non-drakeborgs